app/styles/profile.css

```diff
@@ -101,6 +102,19 @@
 
 .edit-btn {
   padding: 0;
+  margin-top: 0.5rem;
+  color: var(--profile-edit-btn-clr);
+  font-weight: 600;
+  font-size: 1rem;
+}
+
+.profile-edit-button {
+  padding: 8px;
+  z-index: 10;
+  border-radius: 100%;
+  position: absolute;
+  top: 8px;
+  right: 12px;
```

Comment:
why some places it is rem and other px?

Reply:
We have already used px and rem both so used it. although i have changed px to rem.